reinforcement_learning/rlclientlib/logger/eventhub_client.cc

```diff
@@ -26,10 +67,28 @@ namespace reinforcement_learning {
 
   int eventhub_client::init(api_status* status) { return authorization(status); }
 
-  int eventhub_client::send(const std::string& post_data, api_status* status) { return v_send(post_data, status); }
+  int eventhub_client::pop_task(api_status* status) {
+    http_request_task oldest;
+    _tasks.pop(&oldest);
+    const auto status_code = oldest.join();
```

Comment:
Should we check for completed tasks to evict before waiting for oldest task?

Reply:
Since we already get 70MB/Sec with this approach.  Does not makes sense to optimize this now.  Please ignore.